### ISSUE TITLE
Unable to start MCAC with development version of Cassandra

### ISSUE THREAD

**Issue by anthonydugois** [datastax/metric-collector-for-apache-cassandra#79]
Hello everyone,

I am experimenting on performance with the development version of Cassandra (trunk branch) and I would like to monitor my experiments. I am currently unable to start the agent:

```
INFO  [main] 2022-09-02 14:59:56,045 CassandraDaemonInterceptor.java:59 - Starting DataStax Metric Collector for Apache Cassandra 0.3.1
INFO  [main] 2022-09-02 14:59:56,055 ConfigurationLoader.java:62 - Configuration location: file:/opt/agent/config/metric-collector.yaml
INFO  [main] 2022-09-02 14:59:56,159 CollectdController.java:469 - /opt/agent/config/collectd.conf.tmpl
INFO  [main] 2022-09-02 14:59:56,244 UnixSocketClient.java:381 - Connection to collectd established
Exception (java.lang.NoSuchFieldError) encountered during startup: scheduledTasks
java.lang.NoSuchFieldError: scheduledTasks
	at com.datastax.mcac.UnixSocketClient.initCollectdHealthCheck(UnixSocketClient.java:344)
	at com.datastax.mcac.UnixSocketClient.start(UnixSocketClient.java:296)
	at com.datastax.mcac.interceptors.CassandraDaemonInterceptor.intercept(CassandraDaemonInterceptor.java:60)
	at org.apache.cassandra.service.CassandraDaemon.start(CassandraDaemon.java)
	at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:770)
	at org.apache.cassandra.service.CassandraDaemon.main(CassandraDaemon.java:879)
ERROR [main] 2022-09-02 14:59:57,011 CassandraDaemon.java:901 - Exception encountered during startup
java.lang.NoSuchFieldError: scheduledTasks
	at com.datastax.mcac.UnixSocketClient.initCollectdHealthCheck(UnixSocketClient.java:344)
	at com.datastax.mcac.UnixSocketClient.start(UnixSocketClient.java:296)
	at com.datastax.mcac.interceptors.CassandraDaemonInterceptor.intercept(CassandraDaemonInterceptor.java:60)
	at org.apache.cassandra.service.CassandraDaemon.start(CassandraDaemon.java)
	at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:770)
	at org.apache.cassandra.service.CassandraDaemon.main(CassandraDaemon.java:879)
```

Has anyone ever experienced this error at startup?

**Comment by emerkle826** [comment]
I just ran into this myself

**Comment by emerkle826** [comment]
This should be resolved with the latest v0.3.3 release. Make sure to use a 4.1 bundle for Cassandra 4.1 and newer:
https://github.com/datastax/metric-collector-for-apache-cassandra/releases/download/v0.3.3/datastax-mcac-agent-0.3.3-4.1-beta1.tar.gz
or 
https://github.com/datastax/metric-collector-for-apache-cassandra/releases/download/v0.3.3/datastax-mcac-agent-0.3.3-4.1-beta1.zip
